redbot/core/modlog.py

```diff
@@ -294,10 +306,12 @@ async def message_content(self, embed: bool = True):
         else:
             reason = _("**Reason:** Use the `reason` command to add it")
 
-        if self.moderator is not None:
-            moderator = escape_spoilers(f"{self.moderator} ({self.moderator.id})")
-        else:
+        if self.moderator is None:
             moderator = _("Unknown")
+        elif isinstance(self.moderator, int):
+            moderator = f"[Unknown or Deleted User] ({self.moderator})"
```

Comment:
Does this section `[Unknown or Deleted User]` need translating, also?

Reply:
Probably wouldn't hurt, I based it off on similar usage below that was already there but I'll update the PR